server/src/test/java/org/candlepin/policy/js/entitlement/HostedVirtLimitEntitlementRulesTest.java

```diff
@@ -213,17 +308,25 @@ public void exportAllPhysicalZeroBonusPoolQuantity() {
         Entitlement e = new Entitlement(physicalPool, consumer, 10);
         physicalPool.setConsumed(10L);
         physicalPool.setExported(10L);
-        PoolHelper postHelper = new PoolHelper(poolManagerMock, e);
         List<Pool> poolList = new ArrayList<Pool>();
         poolList.add(virtBonusPool);
-        when(poolManagerMock.lookupBySubscriptionId(eq(physicalPool.getSubscriptionId())))
+        ArgumentCaptor<Set> captor = ArgumentCaptor.forClass(Set.class);
+        when(poolManagerMock.lookupBySubscriptionIds(captor.capture()))
             .thenReturn(poolList);
+        when(poolManagerMock.lookupBySubscriptionId(eq("subId"))).thenReturn(poolList);
+        Map<String, Entitlement> entitlements = new HashMap<String, Entitlement>();
+        entitlements.put(physicalPool.getId(), e);
+        enforcer.postEntitlement(poolManagerMock, consumer, entitlements, null);
+
+        @SuppressWarnings("unchecked")
```

Comment:
Move this up to the method signature?  That's where we put the `SuppressWarnings` annotations everywhere else.